tests/align/CMakeLists.txt

```diff
@@ -54,6 +54,33 @@ add_executable (test_align
 # Add dependencies found by find_package (SeqAn).
 target_link_libraries (test_align ${SEQAN_LIBRARIES})
 
+# Add executable for simd tests.
+add_executable(test_align_simd
+               test_align_simd.cpp
+               test_align_simd.h)
+
+# Add dependencies found by find_package (SeqAn).
+target_link_libraries (test_align_simd ${SEQAN_LIBRARIES})
+
+#TODO(rrahn) Enable for windows platforms.
+if (MSVC)
+  # Disable vectorization support on windows for now.
+  # SET (_FLAGS "/arch:AVX")
+else (MSVC)
+  SET (_FLAGS "-march=native")
```

Comment:
we should not set these flags manually in a test. The test-case should just be built and the c++ code decide which actual tests to run.